extensions/amp-twitter/0.1/amp-twitter.js

```diff
@@ -59,7 +59,8 @@ class AmpTwitter extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
-    const iframe = getIframe(this.win, this.element, 'twitter');
+    const iframe = getIframe(this.win, this.element, 'twitter', null,
```

Comment:
let's do `amp-reddit` too, just realized it has the same bug.

Reply:
Done.